sessionresolver/wrapper.go

```diff
@@ -221,10 +221,26 @@ func (sw sessionWrapper) SaveSession(c context.Context, cSession session.Session
 	return sw.repo.InsertOrUpdate(c, cSession.GetId(), cSession)
 }
 
-func (sw sessionWrapper) FreezeCacheVersionsForSession(c context.Context, curSession session.Session) error {
+func (sw sessionWrapper) UnFreezeCacheVersionsForSession(c context.Context, curSession session.Session, action string) error {
 	versions := make(map[string]string)
 	fixedVersions := curSession.GetFixedCacheVersions()
-	versionsForDate, err := sw.repo.GetCacheVersions(c, curSession.GetNow())
+
+	for collection, ver := range fixedVersions {
+		versions[collection] = ver
+	}
```

Comment:
this is unfreeze, shouldnt it:
for ...
  if  !slices.IsStrExist(cacheCollection.LockVersionUpon, filterAction){
     curSession.Delete(cacheCollection)
  }
}